crates/cxx-qt-gen/src/generator/rust/qobject.rs

```diff
@@ -142,12 +158,15 @@ fn generate_qobject_definitions(qobject_idents: &QObjectNames) -> Result<Generat
                 }
             },
         ],
-        implementation: vec![],
+        implementation: base_upcast,
     };
 
     generated
         .cxx_mod_contents
         .append(&mut fragment.cxx_bridge_as_items()?);
+    generated
+        .cxx_mod_contents
```

Comment:
this should be `.cxx_qt_mod_contents` not `.cxx_mod_contents` that should fix the build failures